mongo-fixes/update-dimension-links/main.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"strings"
+	"time"
+
+	"github.com/ONSdigital/dp-dataset-api/models"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+	"github.com/schollz/progressbar/v3"
+)
+
+var (
+	mongoURL string
+)
+
+// MongoID represents instance id
+type MongoID struct {
+	ID bson.ObjectId `bson:"_id"`
+}
+
+// InstanceWithID represents instance with the additional _id from mongo
+// keep in line with `Instance` from $file
```

Comment:
by `$file` I meant the place where this struct came from - you were meant to fill that bit in 😉

Reply:
Ohhhhhhhhhh